code/modules/admin/smites/petrify.dm

```diff
@@ -0,0 +1,11 @@
+/// Turn pur target to stone, forever
+/datum/smite/petrify
+	name = "Petrify"
+
+/datum/smite/petrify/effect(client/user, mob/living/target)
+	. = ..()
+
+	if(!ishuman(target))
+		to_chat(user, span_warning("This must be used on a human subtype."), confidential = TRUE)
+		return
+	target.petrify(statue_timer = INFINITY, save_brain = FALSE)
```

Comment:
linters are failing because this needs to be typecast to human.

Reply:
👍